dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/VoltageRegulators/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ set(MODEL_FILES
     VRProportionalIntegral_INIT.mo
     VRProportionalReactiveFeedback.extvar
     VRProportionalReactiveFeedback.mo
+    VRRemote.extvar
```

Comment:
I am sorry I thought it was a machine regulation when I suggested this place in the library. Given the fact that it is a centralized  control (more as the SVC) it is better to put it in Controls>Voltage as you suggested first

Reply:
Ok I'll change it, no pb!